tests/utils.js

```diff
@@ -0,0 +1,196 @@
+import Promise from 'bluebird'
+import { randomBytes } from 'crypto'
+import { SphereClient } from 'sphere-node-sdk'
+
+import getSphereClientCredentials from '../src/sphere-client-credentials'
+
+export const PROJECT_KEY = process.env.CI === 'true'
+  ? process.env.SPHERE_PROJECT_KEY
+  : process.env.npm_config_projectkey
+
+export async function getClientConfig (projectKey = PROJECT_KEY) {
```

Comment:
You don't need to declare this as an `async` function

Reply:
Async removed here https://github.com/sphereio/sphere-product-type-export/pull/22/commits/0a632e9f8e605d0a558865faec9e5308a357eb5e#diff-74a635be9be46d0f8b20784f5117bb0cR11